src/components/popover/CdrPopover.vue

```diff
@@ -108,10 +108,10 @@
     lastActive = activeElement;
     isOpen.value = true;
     emit('opened', e);
-    nextTick(() => {
+    setTimeout(()=>{
       const tabbables = tabbable(popupEl.value.$el);
       if (tabbables[0]) tabbables[0].focus();
-    });
+    }, 50)
```

Comment:
Turns out we just needed a bit more time

Reply:
Don't we all